### ISSUE TITLE
.bind() and _ are not recognized

### ISSUE THREAD

**Issue by grisevg** [HaxeFoundation/intellij-haxe#72]
In a valid haxe statement `mymethod.bind(5, _, "foo")` both `bind` and `_` are highlighted with an "Unresolved Symbol" error.

TIR: STB-7302

**Comment by Simn** [comment]
`_` is not a keyword, it's simply a valid identifier.

"bind" is a pseudo-field on functions. Note that there's also "match" as a pseudo-field on enum instances.

**Comment by as3boyan** [comment]
So `_` should be added to literalExpression I think...

```
literalExpression ::= LITINT | LITHEX | LITOCT | LITFLOAT
                    | regularExpressionLiteral
                    | 'null' | 'true' | 'false'
                    | functionLiteral
                    | arrayLiteral
                    | objectLiteral
                    | stringLiteralExpression
```

**Comment by EBatTiVo** [comment]
I don't think so.  It's just an identifier.  So, I'd think that *if* we want to encode it in the BNF, we write a new rule like so:
```BNF
predefinedIdentifier :: '_'

identifier ::= ID | predefinedIdentifier
```

Then, when we're resolving it, we write HaxeIdentifier.isPredefined(), and, if so, don't do the normal reference resolution -- or resolve it to some internal construct.  Just a long as we don't show any error, we're fine.  When we're parsing code to show errors, it never actually resolves to anything.

The follow-on is a bit more tricky: It would be nice for completions to show somebody calling through the bound function that they only need to insert arguments for the underscores.  However, if we let the compiler do that, it shouldn't be so hard.

**Comment by as3boyan** [comment]
So you think it would be easier to resolve it if I add new predefinedIdentifier declaration?
